cmd/flux/trace.go

```diff
@@ -44,16 +46,16 @@ var traceCmd = &cobra.Command{
 	Long: `The trace command shows how an object is managed by Flux,
 from which source and revision it comes, and what's the latest reconciliation status.'`,
 	Example: `  # Trace a Kubernetes Deployment
-  flux trace my-app --kind=deployment --api-version=apps/v1 --namespace=apps
+  flux trace -n apps deployment my-app
 
   # Trace a Kubernetes Pod
-  flux trace redis-master-0 --kind=pod --api-version=v1 -n redis
+  flux trace -n redis pod redis-master-0
 
   # Trace a Kubernetes global object
-  flux trace redis --kind=namespace --api-version=v1
+  flux trace namespace redis
 
   # Trace a Kubernetes custom resource
-  flux trace redis --kind=helmrelease --api-version=helm.toolkit.fluxcd.io/v2beta1 -n redis`,
+  flux trace -n redis helmrelease redis`,
```

Comment:
I would still leave an example with the old flags to both document and highlight backwards compatibility with the previous flag set.

Reply:
I've expanded it a bit, wdyt?